design/Frontend-Design.md

```diff
@@ -1 +1,46 @@
-# TBD
\ No newline at end of file
+### Clean code
+
+#### Variables
+Give meaningful names to variables
+
+#### Functions
+1. Function name should describe what it does.
+2. A function should do only one thing.
+3. Limit the amount of function parameters, preferably two. if you need more you can send the parameters inside an object.
+4. If you find yourself writing the same code more than once, even if its a little bit different, try to generalize it in to a function.
+5. Functions should be pure
```

Comment:
[Sand]
Maybe we should add a small explanation or a reference to what `pure` means..

Reply:
Pure function is the life